tests/e2e/lib.sh

```diff
@@ -439,6 +439,45 @@ setup_automation_flavor_e2e_cluster() {
     fi
 }
 
+# When working as expected it takes less than one minute for the API server to
+# reach ready. Often times out on OSD. If this call fails in CI we need to
+# identify the source of pull/scheduling latency, request throttling, etc.
+# I tried increasing the timeout from 5m to 20m for OSD but it did not help.
+wait_for_central_db() {
+    info "Waiting for Central DB to start"
+
+    start_time="$(date '+%s')"
+    max_seconds=300
+
+    while true; do
+        central_db_json="$(kubectl -n stackrox get deploy/central-db -o json)"
+        replicas="$(jq '.status.replicas' <<<"$central_db_json")"
+        ready_replicas="$(jq '.status.readyReplicas' <<<"$central_db_json")"
+        curr_time="$(date '+%s')"
+        elapsed_seconds=$(( curr_time - start_time ))
+
+        # Ready case
+        if [[ "$replicas" == 1 && "$ready_replicas" == 1 ]]; then
+            sleep 30
```

Comment:
Why do we need this if replicas are ready?

Reply:
@janisz I mirrored (actually copied) this from the existing wait_for_api.  So I assumed that was there for reasons.  I could take it out if everyone likes, but I don't think it does any harm.